src/features/moviesPage/moviePage/BigMovieTile/index.js

```diff
@@ -0,0 +1,23 @@
+import { imagesApi } from "../../../../core/API";
+import { BackDropWrapper, BackGround, Backdrop, Information, Rate, Rating, Star, Title, Votes } from "./styled";
+import star from "../../../../common/images/star.svg"
+
+const BigMovieTile = ({ poster, title, rate, voteCount }) => (
+    <BackGround>
+        <BackDropWrapper>
+            <Information>
+                <Title>{title}</Title>
+                <Rating>
+                    <Rate>
+                        <Star src={star} />
+                        {rate.toFixed(1)} <Votes>/ 10</Votes>
+                    </Rate>
+                    <Votes>{voteCount} votes</Votes>
+                </Rating>
+            </Information>
+            <Backdrop src={`${imagesApi}/original${poster}`}></Backdrop>
```

Comment:
Please remember about alts - they have quite a significant impact on the SEO of the page